main.tf

```diff
@@ -28,7 +28,7 @@ resource "aws_s3_bucket" "lambda_newrelic_resource" {
 
 resource "aws_s3_bucket_object" "newrelic_log_ingestion_zip" {
   bucket = aws_s3_bucket.lambda_newrelic_resource.id
-  key    = "newrelic-log-ingestion.zip"
+  key    = "newrelic-log-ingestion-2.3.5.zip"
```

Comment:
is this correct? Or should point to zip file without version?

Reply:
this is the name in the s3 bucket, and when the name is not changing this cloudformation stack will never update so I have decided to write as the key the version of the zip file